tests/sentry/incidents/endpoints/test_project_alert_rule_index.py

```diff
@@ -38,7 +38,7 @@ def test_simple(self):
         alert_rule = self.create_alert_rule()
 
         self.login_as(self.user)
-        with self.feature("organizations:incidents"):
+        with self.feature(["organizations:incidents", "organizations:incidents-performance"]):
```

Comment:
Worth adding a test to verify that the alert rule doesn't appear if you don't have performance?